content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+The following fields are returned if valued:
+
+* [Encounter id](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.id){:target="_blank"}
+* [Encounter identifier (eg: FIN)](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.identifier){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.status){:target="_blank"}
+* [Class](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.class){:target="_blank"}
+* [Type](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.type){:target="_blank"}
+* [Priority](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.priority){:target="_blank"}
+* [Subject](http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.subject){:target="_blank"}
+* Participant:
```

Comment:
Should we consider adding link for the parent element?
http://hl7.org/fhir/encounter-definitions.html#Encounter.participant

Reply:
talked with kathy, she said we should: https://github.com/cerner/fhir.cerner.com/pull/238/commits/223ff452692cfe331e09c3b190ee83e615721033#diff-6590262e197180cc09f6ad70a2a01a5bL23

Reply:
+1